packages/connectivity/src/scp-cf/cache.ts

```diff
@@ -104,22 +94,11 @@ function isExpired<T>(item: CacheEntry<T>): boolean {
 }
 
 function inferExpirationTime(
-  expirationTime: DateInputObject | undefined
+  expirationTime: number | undefined
```

Comment:
```suggestion
  validityTimeInMs: number | undefined
```
[req] Add `InMs` to indicate unit of measurement + Consistent naming, parameter should be called `validityTimeInMs`

Reply:
thank you for your comment. I changed the parameter name as your comment for consistent naming.